ocs_ci/templates/ocs-deployment/local-volume-discovery.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: local.storage.openshift.io/v1alpha1
+kind: LocalVolumeDiscovery
+metadata:
+  name: auto-discover-devices
+spec:
+  nodeSelector:
+    nodeSelectorTerms:
+      - matchExpressions:
+          - key: kubernetes.io/hostname
+            operator: In
+            values:
+              - PLACEHOLDER
```

Comment:
Add an empty line to the end of these yaml files

Reply:
Done